impact.ipynb

```diff
@@ -19,7 +19,15 @@
      "parameters"
     ]
    },
-   "outputs": [],
+   "outputs": [
```

Comment:
this is the critical piece here: we have a github action check that there are no output cells for all jupyter notebooks checked into source control.